units/patch_intent.go

```diff
@@ -493,36 +473,15 @@ func authAndFetchPRMergeBase(ctx context.Context, patchDoc *patch.Patch, require
 			"pr_number":    patchDoc.GithubPatchData.PRNumber,
 		})
 		return false, err
-	}
 
-	commits, _, err := client.PullRequests.ListCommits(ctx, patchDoc.GithubPatchData.BaseOwner,
-		patchDoc.GithubPatchData.BaseRepo, patchDoc.GithubPatchData.PRNumber, nil)
-	if err != nil {
-		return isMember, err
-	}
-	if len(commits) == 0 {
-		return isMember, errors.New("No commits received from github")
-	}
-	if commits[0].SHA == nil {
-		return isMember, errors.New("hash is missing from pull request commit list")
 	}
 
-	commit, _, err := client.Repositories.GetCommit(ctx,
-		patchDoc.GithubPatchData.BaseOwner, patchDoc.GithubPatchData.BaseRepo, *commits[0].SHA)
+	hash, err := thirdparty.GetPullRequestMergeBase(ctx, githubOauthToken, patchDoc.GithubPatchData.BaseOwner, patchDoc.GithubPatchData.BaseRepo, patchDoc.GithubPatchData.PRNumber)
 	if err != nil {
 		return isMember, err
```

Comment:
might be useful for future troubleshooting to `grip.Error` the error here with some extra metadata